.gitignore

```diff
@@ -1,5 +1,6 @@
 .gradle
 /local.properties
+/.idea
```

Comment:
The .gitignore is the default Android Studio .gitignore.
Why should .idea be excluded completely?